posts/2021/04/pytorch-tensoriterator-internals-update.md

```diff
@@ -0,0 +1,337 @@
+<!--
+.. title: PyTorch TensorIterator Internals (TensorIteratorConfig Update)
```

Comment:
rendering of this title comes out awkwardly: 

<img width="569" alt="image" src="https://user-images.githubusercontent.com/98330/114194420-c1ee0600-994f-11eb-87c8-583e9aa43fbd.png">

`TensorIteratorConfig` is also too specific for being in the title I think. How about _"PyTorch TensorIterator internals - 2021 update"_?

Reply:
I like that title, I've changed it